loader/checkpoint.go

```diff
@@ -346,6 +381,16 @@ func (cp *RemoteCheckPoint) GenSQL(filename string, offset int64) string {
 	return sql
 }
 
+// UpdateOffset implements CheckPoint.UpdateOffset
+func (cp *RemoteCheckPoint) UpdateOffset(filename string, offset int64) {
+	fields := strings.Split(filename, ".")
```

Comment:
How about use job.schema, job.table instead?

Reply:
I've thinked of that, and found `job.schema` is target schema, here we need source schema. Bad variable name or lack comments 😂 

Hope we could replace it to lightning soon